cmd/options.go

```diff
@@ -10,6 +10,7 @@ import (
 func AddDiffOptions(f *pflag.FlagSet, o *diff.Options) {
 	f.BoolP("suppress-secrets", "q", false, "suppress secrets in the output")
 	f.BoolVar(&o.ShowSecrets, "show-secrets", false, "do not redact secret values in the output")
+	f.BoolVar(&o.DecodeSecrets, "decode-secrets", false, "decode secret values in the output")
```

Comment:
👋 Quick one from me!
I tend to prefer avoiding combinatinal explosion of flags.
This seems to work only when you've also specified show-secrets.
Can we name this like "show-secrets-decoded" that works alone?
It will be mutually exclusive to the existing "show-secrets".

Reply:
@mumoshu good idea.

Reply:
@mumoshu done